test/unnest.js

```diff
@@ -26,4 +27,38 @@ describe('unnest', function() {
     assert.deepEqual(R.unnest([[], [], []]), []);
     assert.deepEqual(R.unnest([]), []);
   });
+
+  it('is equivalent to R.chain(R.identity)', function() {
+    function Nothing() {
+      if (!(this instanceof Nothing)) {
+        return new Nothing();
+      }
+    }
+    Nothing.prototype.chain = function() {
+      return this;
+    };
+    Nothing.prototype.toString = function() {
+      return 'Nothing()';
+    };
+
+    function Just(x) {
+      if (!(this instanceof Just)) {
+        return new Just(x);
+      }
+      this.value = x;
+    }
+    Just.prototype.chain = function(f) {
+      return f(this.value);
+    };
+    Just.prototype.toString = function() {
+      return 'Just(' + R.toString(this.value) + ')';
+    };
+
```

Comment:
Or just replace lines 32 - 55 with 

```js
var Maybe = require('./shared/Maybe');
```

:smile: 

See [`test/commute.js`][co] or [`test/lift.js`][li] for examples.

  [co]: https://github.com/ramda/ramda/blob/292e081fff69ad9f6ba39e7295de032c6ff4ee8a/test/commute.js#L2
  [li]: https://github.com/ramda/ramda/blob/292e081fff69ad9f6ba39e7295de032c6ff4ee8a/test/lift.js#L6